### ISSUE TITLE
Upload Project functionality

### ISSUE THREAD

**Issue by SPraveenKumar-spk** [SUGAM-ARORA/UniCollab#22]
Under this :

1. way to upload the projects.
2. Fetch the uploaded projects in feed section.

@SUGAM-ARORA  Please assign this to me under GSSOC24. I would like to contribute to this. Also please label it GSSOC24 and its appropriate level of difficulty.

**Comment by SUGAM-ARORA** [comment]
@SPraveenKumar-spk Since you have been already assigned an issue #21 so I won't be able to assign this to you at the moment. But after completion of previous issue I will assign this to you.

**Comment by VanshGoyal000** [comment]
Hey i Think i can do this Please assign it to me!

**Comment by SUGAM-ARORA** [comment]
@VanshGoyal000 Issue assigned to you

**Comment by VanshGoyal000** [comment]
> @VanshGoyal000 Issue assigned to you

in this no backend written till now , so i have to make structure for backend and seprate frontend and backend folder in root directory so can I?

**Comment by SUGAM-ARORA** [comment]
> > @VanshGoyal000 Issue assigned to you
> 
> in this no backend written till now , so i have to make structure for backend and seprate frontend and backend folder in root directory so can I?

Yes you can. But make sure that in that process any existing thing doesn't break

**Comment by VanshGoyal000** [comment]
Before this i want login signup feature because in this you need to have a particular user for uploading file and showing them in feed

**Comment by SUGAM-ARORA** [comment]
> Before this i want login signup feature because in this you need to have a particular user for uploading file and showing them in feed

Absolutely. Till then if you want I can assign you some other issue.
